setup.py

```diff
@@ -64,10 +64,11 @@ def long_description():
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: Implementation :: CPython',
         'Programming Language :: Python :: Implementation :: PyPy',
     ],
-    python_requires='>=3.5, <3.8',
+    python_requires='>=3.5, <3.9',
```

Comment:
I think `<=3.8` is more symmetric in relation to `>=3.5`.

Reply:
That seems reasonable.

Still pending on a release of `zulip` for now, though.

Reply:
Fixed in current version.